docs/configuring.md

```diff
@@ -703,6 +703,47 @@ questions with default answers.
     copier -fd 'user_name=Manuel Calavera' copy template destination
     ```
 
+!!! info
+
+    The `--data` option is mutually exclusive with [`--data-file`](#data_file) option.
+
+### `data_file`
+
+-   Format: `str`
+-   CLI flags: `--data-file`
+-   Default value: N/A
+
+As an alternative you can also pass the path to a YAML file that contains your data.
```

Comment:
Just in case some reader lands here directly without reading the previous chapter before:

```suggestion
As an alternative to [`--data`][data] you can also pass the path to a YAML file that contains your data.
```